nmigen_boards/colorlight_5a_75b_7_0.py

```diff
@@ -0,0 +1,110 @@
+import os
+import subprocess
+
+from nmigen.build import *
+from nmigen.vendor.lattice_ecp5 import *
+from nmigen_boards.resources import *
+
+
+__all__ = ['Colorlight5a75b70Platform']
+
+
+class Colorlight5a75b70Platform(LatticeECP5Platform):
+    device = 'LFE5U-25F'
+    package = 'BG256'
+    default_clk = 'clk25'
+    speed = '8'
+
+    resources = [
+        Resource('clk25', 0, Pins('P6'), Clock(25e6), Attrs(GLOBAL=True, IO_STANDARD='LVCMOS33')),
+
+        *LEDResources(pins='P11', attrs=Attrs(IO_STANDARD='LVCMOS33')),
+        Resource('user_led', 0, PinsN('P11', dir='o'), Attrs(IO_STANDARD='LVCMOS33')),
+
+        *ButtonResources(pins='M13', attrs=Attrs(IO_STANDARD='LVCMOS33')),
+        Resource('user_btn', 0, PinsN('M13'), Attrs(IO_STANDARD='LVCMOS33')),
+
+        # available in the J19 connector (rx = btn, tx=led)
+        UARTResource(0,
+            rx='P11', tx='M13',
+            attrs=Attrs(IO_STANDARD='LVCMOS33')
+        ),
+
+        # W25Q32JV
+        #SPIResource(0,
```

Comment:
Please update the definition to be up-to-date (use COPI etc) with a comment about `USRMCLK`. At the moment it is not possible to request this resource but this will be fixed.